server/src/main/java/org/elasticsearch/gateway/DanglingIndicesState.java

```diff
@@ -55,19 +56,35 @@
 
     private static final Logger logger = LogManager.getLogger(DanglingIndicesState.class);
 
+    public static final Setting<Boolean> AUTO_IMPORT_DANGLING_INDICES_SETTING = Setting.boolSetting(
```

Comment:
Let's deprecate this setting right away, so that we can remove it in 8.0 if we want